src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -18,6 +18,73 @@ public static void OpenMyStore()
             }
         }
 
+#if netcoreapp11
+        [Fact]
+        public static void Constructor_DefaultStoreName()
+        {
+            using (X509Store store = new X509Store(StoreLocation.CurrentUser))
+            {
+                Assert.Equal("My", store.Name);
+            }
+        }
+
+        [Fact]
+        public static void Constructor_DefaultStoreLocation()
+        {
+            using (X509Store store = new X509Store(StoreName.My))
+            {
+                Assert.Equal(StoreLocation.CurrentUser, store.Location);
+            }
+
+            using (X509Store store = new X509Store("My"))
+            {
+                Assert.Equal(StoreLocation.CurrentUser, store.Location);
+            }
+        }
+
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [Fact]
+        public static void Constructor_StoreHandle()
+        {
+            X509Store store1;
+            using (store1 = new X509Store(StoreName.My, StoreLocation.CurrentUser))
+            {
+                store1.Open(OpenFlags.ReadOnly);
+                int certCount1;
+
+                using (var coll = new ImportedCollection(store1.Certificates))
+                {
+                    certCount1 = coll.Collection.Count;
+                    Assert.True(certCount1 >= 0);
+                }
+
+                using (X509Store store2 = new X509Store(store1.StoreHandle))
+                {
+                    using (var coll = new ImportedCollection(store2.Certificates))
+                    {
+                        int certCount2 = coll.Collection.Count;
+                        Assert.Equal(certCount1, certCount2);
```

Comment:
This test is subject to an external race condition (anything being added to CU\My); though we probably have similar problems elsewhere.